pccommon/tests/config/test_render_config.py

```diff
@@ -5,21 +5,21 @@
 
 multi_assets = DefaultRenderConfig(
     assets=["data1", "data2"],
-    render_params={"colormap_name": "terrain", "rescale": [-1000, 4000]},
+    render_params={"colormap_name": "terrain", "rescale": "-1000,4000"},
```

Comment:
FYI @pholleway and @gadomski this is the change to the render config I was trying to explain. `[1, 2]` -> `["1,2"]`. This allows us to handle multiple bands.